<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPlayerDialog</class>
 <widget class="QDialog" name="AddPlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Player</string>
  </property>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>16</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>83</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>461</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="positionLabel">
          <property name="text">
           <string>Position:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="positionComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>QB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>WR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>K</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DST</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="playerLabel">
          <property name="text">
           <string>Player:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="playerLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="submitButton">
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
